projects/laji/src/app/shared-modules/document-viewer/document-annotation/document-annotation.component.ts

```diff
@@ -429,10 +429,9 @@ export class DocumentAnnotationComponent implements AfterViewInit, OnChanges, On
   }
 
   private move() {
-    this.document = this.result[this.indexPagination].document;
-    this.result.findIndex(i => (i.fullURL === this.highlight || i.unit?.unitId === this.highlight));
-    this.uri = (this.result[this.indexPagination].documentId || this.result[this.indexPagination].document?.documentId);
-    this.highlight = (this.result[this.indexPagination].fullURL || this.result[this.indexPagination].unit?.unitId);
+    this.documentOrImage = this.result[this.indexPagination];
```

Comment:
good if this can be done :)

Now though the documentOrImage would hold when result is a list of documents a value that would be something like this `{document: {documentId: 'JA.1', ...}}` Previously this picked `.document` from the results with the wanted index so that it would have hold something like this: `{documentId: 'JA.1', ...}` .  Can you confirm that this is correct and I'll test this more thoroughly

Reply:
@vixriihi i can confirm that it's working at least in Observation/annotations and identify pages. Result in observation/annotations is an object that contains other object "document" and in indentify is a single object which contains properties "documentId" and "FullURL"

Reply:
`this.document` based on the view should only hold document information I added a local variable to this method so that it would not reuse the `this.document` to something that it was not indented. Reason why it works is that right after in this method it calls `updateDocument()` which in turn updates that variable back to document